rust-toolchain

```diff
@@ -0,0 +1 @@
+1.58.1
```

Comment:
What's this file?

Reply:
It will cause cargo to always build on 1.58.1